content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -39,10 +43,17 @@ LoadBalancer, or with dynamic PersistentVolumes.
 For both methods you need this infrastructure:
 
 - Three machines that meet [kubeadm's minimum
-  requirements](/docs/setup/independent/install-kubeadm/#before-you-begin) for
+<<<<<<< HEAD:content/en/docs/setup/production-environment/tools/independent/high-availability.md
```

Comment:
This looks like a merge left in by mistake.

Reply:
Thank you it was a miss. Fixed it now.